bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -118,7 +118,8 @@
     public static final ChannelType SYSTEM_CHANNEL_LOW_BATTERY = ChannelTypeBuilder
             .state(SYSTEM_CHANNEL_TYPE_UID_LOW_BATTERY, "Low Battery", CoreItemFactory.SWITCH)
             .withCategory("LowBattery")
-            .withStateDescriptionFragment(StateDescriptionFragmentBuilder.create().withReadOnly(true).build()).build();
+            .withStateDescriptionFragment(StateDescriptionFragmentBuilder.create().withReadOnly(true).build())
+            .withTags(List.of("LowBattery", "Level")).build();
```

Comment:
I would use "LowBattery" and "Energy" in this case:

```suggestion
            .withTags(List.of("LowBattery", "Energy")).build();
```

Reply:
Ok